_ingest-pipelines/processors/ml-inference.md

```diff
@@ -94,6 +113,31 @@ PUT /_ingest/pipeline/ml_inference_pipeline
 ```
 {% include copy-curl.html %}
 
+For externally hosted models, in a Predict API request, all fields are usually nested inside the `parameters` object:
```

Comment:
"For a Predict API request in an externally hosted model"? The back-to-back phrases are a little awkward.